src/Microsoft.AspNetCore.Razor.Language/DefaultDirectiveSyntaxTreePass.cs

```diff
@@ -44,7 +44,9 @@ public override void VisitDirectiveBlock(DirectiveChunkGenerator chunkGenerator,
                 if (_nestedLevel > 0)
                 {
                     var directiveStart = block.Children.First(child => !child.IsBlock && ((Span)child).Kind == SpanKindInternal.Transition).Start;
-                    var errorLength = /* @ */ 1 + SectionDirective.Directive.Directive.Length;
```

Comment:
I'm a fan of leaving these cases as `/* ... */`. They're informational (in this case, saying that the `1` corresponds with the `@` without making the line harder to read. Same for elsewhere.